pkg/manager/member/tikv_upgrader.go

```diff
@@ -149,6 +149,12 @@ func (u *tikvUpgrader) Upgrade(meta metav1.Object, oldSet *apps.StatefulSet, new
 			return nil
 		}
 
+		if len(podOrdinals) < 2 {
```

Comment:
What about moving the check to L79 and just check the `*oldSet.Spec.Replicas`?

Reply:
Ok, not clear for meam of the `partition` of statefulset before.